### DISCUSSION TITLE
0.4.4版本打开白屏

### DISCUSSION THREAD

**Discussion by zlf179** [qier222/YesPlayMusic#1326] [Q&A] [answered]
每一页都是白屏 加载不出任何内容 关掉路由器端去广告也无效

![TIM图片20220220194348](https://user-images.githubusercontent.com/26280126/154861234-c7ea7721-fe02-4782-8646-0945743960f5.png)
win11 客户端  `resetApp()`命令也无效

**Comment by pan93412** [comment, marked as answer]
假如你有啟用 UnblockNeteaseMusic，請將其關閉。

**Reply by zlf179** [reply]
确实 关掉了 就好了

**Reply by hsiayuchih** [reply]
开了这个后听歌听着听着就会弹出这个，关了后过一会还会再次弹出
![QQ截图20220221133813.png](https://tva1.sinaimg.cn/large/6461cddagy1gzl3q45apkj20fm0c7n3y.jpg)
